examples/source/components/amp-dynamic-css-classes.html

```diff
@@ -3,27 +3,27 @@
 <head>
   <meta charset="utf-8">
   <title>amp-dynamic-css-classes</title>
-  <!-- #### Introduction 
-     AMP's dynamic CSS classes provided by the [amp-dynamic-css-class](https://www.ampproject.org/docs/reference/components/presentation/amp-dynamic-css-classes) enable boolean logic for a handful of conditions 
+  <!-- #### Introduction
```

Comment:
Could you do me favour and replace `s/####/##/`. Looks like I forgot to do this during the migration.

Reply:
Sure 👌